src/Qbead.h

```diff
@@ -150,20 +257,20 @@ class Qbead {
   const uint8_t ix, iy, iz;
   const bool sx, sy, sz;
   float rbuffer[3];
-  float x, y, z, rx, ry, rz; // filtered and raw acc, in units of g
-  float t_acc, p_acc;        // theta and phi according to gravity
```

Comment:
These should not be removed. They have little to do with the `State` object. Sure, someone can decide to use the IMU to update the State object, but that is not something we should hardcode.

These are simply measurement results and readIMU should be updating them and having them available to users to do whatever they want with them.